vignettes/S4VectorOverview.Rmd

```diff
@@ -0,0 +1,346 @@
+---
+title: "**An Overview of the S4Vectors package**"
+author: 
+- name: "Patrick Aboyoun"
+- name: "Michael Lawrence"
+- name: "Hervé Pagès"
+date: "February 2018; compiled: `r format(Sys.time(), '%B %d , %Y')`"
+package: S4Vectors
+vignette: >
+  %\VignetteIndexEntry{An Overview of the S4Vectors package}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteKeywords{Vector,Hits,Rle,List,DataFrame}
+  %\VignettePackage{S4Vectors}
+output:
+  BiocStyle::html_document:
+        number_sections: true
+        toc: true
+        toc_depth: 4
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+# Introduction
+
+The
+[*S4Vectors*](http://bioconductor.org/packages/release/bioc/html/S4Vectors.html)

```

Comment:
Replace `[*S4Vectors*](http://bioconductor.org/packages/release/bioc/html/S4Vectors.html)` with 

```
`r Biocpkg("S4Vectors")`
```

You'll need to do this for all instances of S4Vectors and other Bioc packages.